src/main/java/core/basesyntax/strategy/impl/SupplyOperationHandler.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.enums.Operation;
+import core.basesyntax.strategy.QuantityIncrementer;
+import java.util.Map;
+
+public class SupplyOperationHandler
+        extends AbstractOperationHandler
+        implements QuantityIncrementer {
+    public SupplyOperationHandler() {
+        super(Operation.SUPPLY);
+    }
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the constructor (line 11).